tools/ncbi_datasets/macros.xml

```diff
@@ -51,7 +51,7 @@
         </conditional>
     </xml>
     <xml name="chromosomes">
-        <param argument="--chromosomes" type="text" value="all" label="Limit chromosomes to a comma-delimited list of chromosomes">
+        <param argument="--chromosomes" type="text" label="Limit chromosomes to a comma-delimited list of chromosomes">
```

Comment:
So the default value here would be and empty string and the templated command line fragment would be `--chromosomes ''` ... does this make sense, or should we skip `--chromosomes` if the user hasn't entered anything here ?

Reply:
It should be skipped.  I changed the command to 

```
#if $filters.chromosomes:
--chromosomes '$filters.chromosomes'
#end if
```

Reply:
Sorry, completely missed that. Looks good then!